mods/dark-menus.wh.cpp

```diff
@@ -0,0 +1,63 @@
+// ==WindhawkMod==
+// @id              dark-menus
+// @name            Dark mode context menus
+// @description     Enables dark mode for all win32 menus.
+// @version         1.0
+// @author          Mgg Sk
+// @github          https://github.com/MGGSK
+// @include         *
+// ==/WindhawkMod==
+
+// ==WindhawkModReadme==
+/*
+# Dark mode context menus
+Forces dark mode for all win32 context menus to create a more consistent UI.
+*/
+// ==/WindhawkModReadme==
+
+#include <minwindef.h>
+#include <winerror.h>
+
+enum AppMode
+{
+	Default,
+	AllowDark,
+	ForceDark,
+	ForceLight,
+	Max
+};
+
+using FlushMenuThemes_T = void (WINAPI *)();
+using SetPreferredAppMode_T = HRESULT (WINAPI *)(AppMode appMode);
+
+FlushMenuThemes_T FlushMenuThemes;
+SetPreferredAppMode_T SetPreferredAppMode;
+
+//Applies the theme to all menus.
+HRESULT ApplyTheme()
+{
+    FlushMenuThemes();

```

Comment:
Shouldn't `FlushMenuThemes()` be called after `SetPreferredAppMode`?

Reply:
This is the correct way to apply a menu theme because the SetPreferredAppMode function only works when the menu has no theme applied and the FlushMenuThemes function removes all themes from the menu.